### DISCUSSION TITLE
Possible to add liked trakt.tv lists to individual collecions ?

### DISCUSSION THREAD

**Discussion by gitttt54** [meisnate12/Plex-Meta-Manager#637] [Q&A] [answered]
Hello,

I'm aware, I can add several trakt lists into one single collection, like:
```
  Spooky Season Approaches:
    template: {name: Holiday, poster: /config/posters/paintings/halloween.jpg}
    trakt_list:
      - https://trakt.tv/users/kairbear08/lists/halloween
      - https://trakt.tv/users/philrivers/lists/dreadit-s-top-100-horror-films-2018-edition-r-horror
      - https://trakt.tv/users/galacticboy/lists/horror-in-past-90-days
    summary: A mix of Halloween themed movies and just movies that are fun to watch around Halloween.
    sync_mode: append
```

But **how would I add all my liked lists on trakt.tv to separate collections ?**
    (without adding each of them statically in the configfile.yml !!)

Is this possible?

It would ease my life, because I have about 80 liked lists and its easy to add or remove lists via the trakt.tv webgui.

But its not that easy to have that list of lists being in sync with a config-file.yml, IMHO.

**Comment by meisnate12** [comment, marked as answer]
This is a great addition to a feature I'm working on be on the lookout for something to help with this in 1.16.0

**Reply by gitttt54** [reply]
So is this possible now?  (I installed version 1.17.1)

I found something in the wiki => https://metamanager.wiki/en/latest/metadata/dynamic.html#trakt-liked-lists

but unfortunately could not make it work, as I expected it to work ;-)

(I expected PMM to create in plex all the lists I have liked on trakt. Maybe this is a wrong expectation...)

Here is my config and I have no errors after runnung PMM, but also have no collections created from trakt liked lists (nor popular people).

What am I missing in my config?

I tried to peek into some configs in the config repo, but did not find any which would help/work.

```
libraries:
  Filme:
    library_name: Filme
    operations:
      delete_collections_with_less: 2
    dynamic_collections:
      TMDb Collections:              # This name is the mapping name
        type: tmdb_collection
        remove_suffix: Collection
        remove_prefix: The
      TMDb Popular People:              # This name is the mapping name
        type: tmdb_popular_people
        data: 10
      Trakt Liked Lists:              # This name is the mapping name
        type: trakt_liked_lists
        data: me
      Trakt User Lists:
        type: trakt_people_lists
        data:
        - https://trakt.tv/users/ash9001/lists/all-time-top-actors

```

**Reply by gitttt54** [reply]
no one?

**Reply by meisnate12** [reply]
dynamic collections go in a metadata file not in your config file
